wafflehaus/edit_response/__init__.py

```diff
@@ -52,27 +52,57 @@ def _replace_lookup(self, replace_str):
             return {}
         return replace_str
 
+    def _foreach(self, conditional, data):
+        if not isinstance(data, list):
+            return data
+        if not all(isinstance(item, dict) for item in data):
```

Comment:
Side Note: I believe this runs `all()` on a generator. So even if you run `all(x for x in (0,-1,True))`, you get False.